### PR TITLE
PMM-3865 update pmm2-server-release job

### PR DESCRIPTION
**Pull request by hors**

### CODE REVIEW

File: pmm/pmm2-server-release.groovy [hunk 1]
```diff
@@ -93,8 +93,8 @@ pipeline {
                     sh """
                         ssh -o StrictHostKeyChecking=no -i ${KEY_PATH} ${USER}@repo.ci.percona.com "
                             rsync -avt --bwlimit=50000 --delete --progress --exclude=rsync-* --exclude=*.bak \
-                                /srv/repo-copy/pmm/ \
-                                10.10.9.209:/www/repo.percona.com/htdocs/pmm/
+                                /srv/repo-copy/pmm2-components/yum/release \
+                                10.10.9.209:/www/repo.percona.com/htdocs/pmm2/
```
Comment (hors) [inline pmm/pmm2-server-release.groovy]: Is it correct path?

File: pmm/pmm2-server-release.groovy [hunk 2]
```diff
@@ -5,22 +5,22 @@ library changelog: false, identifier: 'lib@master', retriever: modernSCM([
...
     parameters {
         choice(
-            choices: 'laboratory\npmm-hotfix-laboratory',
-            description: 'publish pmm-server packages from regular(laboratory) or hotfix(pmm-hotfix-laboratory) repository',
+            choices: 'laboratory',
+            description: 'publish pmm2-server packages from regular(laboratory) repository',
             name: 'UPDATER_REPO')
         string(
-            defaultValue: 'perconalab/pmm-server:dev-latest',
+            defaultValue: 'perconalab/pmm-server:2',
```
Comment (hors) [inline pmm/pmm2-server-release.groovy]: What default tag will we use for pmm-server docker?

Comment (EvgeniyPatlan) [reply pmm/pmm2-server-release.groovy]: for pmm2 - dev-latest

Comment (hors) [reply pmm/pmm2-server-release.groovy]: @EvgeniyPatlan  in this case 'perconalab/pmm-server:1' will be for pmm?

**Pull request merged**